.ci/lanl/gitlab-darwin-ci.yml

```diff
@@ -0,0 +1,114 @@
+variables:
+  SCHEDULER_PARAMETERS: "-pgeneral -t 1:00:00 -N 1 --ntasks-per-node=16"
+  GIT_STRATEGY: clone
+  NPROCS: 4
+
+stages:
+  - build
+  - test
+
+build:intel:
+  stage: build
+  tags: [darwin-slurm-shared]
+  script:
+    - module load intel
+    - git submodule update --init
+    - ./autogen.pl
+    - ./configure CC=icc FC=ifort CXX=icpc --prefix=$PWD/install_test --with-libevent=internal
+    - make -j 8 install
+    - make check
+    - export PATH=$PWD/install_test/bin:$PATH
+    - cd examples 
+    - make
+  variables:
+    SCHEDULER_PARAMETERS: "-pgeneral -t 1:00:00 -N 1 --ntasks-per-node=16"
+  artifacts:
+    name: "$CI_JOB_NAME-$CI_COMMIT_REF_NAME"
+    untracked: true
+    paths:
+      - examples
+      - install_test
+    expire_in: 1 week
+  only:
```

Comment:
Does this `only:` option avoid all commits and only trigger on "gitlab scheduled" pipeline runs?

Reply:
that's how i understand it yes.